infrastructure_source_code.md

```diff
@@ -0,0 +1,44 @@
+# DataWorks Open Source Infrastructure Source Code Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of information can and cannot be included in infrastructure code that is made publicly available.
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source infrastructure code policy which encompasses all DWP public GitHub repositories. Some useful [DWP guidance on creating open source products](https://confluence.service.dwpcloud.uk/pages/viewpage.action?title=Guidance+on+creating+open+source+products&spaceKey=EN) currently exists, but it has not yet been made open-source.
+
+## Scope
+
+This policy covers infrastructure code. Please refer to the [index](README.md#index) for links to other related polices.
+
+## General Approach
+
+Infrastructure code should be made open wherever it is appropriate to do so:
+
+This approach makes code more readily accessible to a broad target audience, so that they can be more easily used for their intended purpose;
+1. This approach enables and encourages re-use of common standards across the DWP and other Government departments, which in turn makes collaboration easier. It also enables other parties to feed back improvements, which benefits everyone.
+1. This policy is in line with the spirit of the  [Government Digital Service (GDS) guidelines on making information open-source](https://gds-operations.github.io/guidelines/).
+
+## Security Restrictions
+
+Reference information which could be used to support targeted attacks on DataWorks or other DWP resources must not be published in open repositories.
+
+Sensitive reference information can take several forms. A non-exhaustive list of these is provided below:
```

Comment:
Things like whitelists and blacklists (IP whitelists/blacklists, site whitelists/blacklists, etc) can potentially provide valuable information/feedback to an attacker, and should be excluded. Happy to discuss some of the reasoning for this offline.

Reply:
Are they are already covered by the fact that the IPs are classed as sensitive? Or do you mean the mere mention of the existence of any IP whitelists should be classed as sensitive?

Reply:
The existence of whitelists is to be expected, and not sensitive. The contents of some whitelists will be.